legend-engine-pure-code-compiled-core/src/main/resources/core/pure/test/testable.pure

```diff
@@ -28,6 +28,13 @@ Class <<typemodifiers.abstract>> meta::pure::test::AtomicTest extends Test
     assertions  : TestAssertion[1..*];
 }
 
+Class <<typemodifiers.abstract>> meta::pure::test::TestBatch
+{
+    id         : String[1];
+    batchId    : Integer[0..1];
```

Comment:
I believe we also discussed to keep batchId mandatory?

Reply:
Thx for all your guidance on this one, Hardik!

If this is the main objection, can we push as-is? I will submit a follow-up PR this afternoon (NY time) correcting multiplicity.

If there are bigger issues, let me know and we will wait for Mon.

Reply:
Testing this change now.